commcare_connect/opportunity/visit_import.py

```diff
@@ -191,7 +191,7 @@ def _bulk_update_payments(opportunity: Opportunity, imported_data: Dataset) -> P
         raise ImportException(f"{len(invalid_rows)} have errors", invalid_rows)
 
     seen_users = set()
-    access_ids = set()
+    payment_ids = set()
```

Comment:
nit: this doesn't have to be a set anymore since the payments are all newly created

Reply:
Removed the set. https://github.com/dimagi/commcare-connect/pull/221/commits/87882d08db84332c75bc70f97c662802be3686e5